charts/memcached/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: memcached
 description: Memcached is an in-memory key-value store for small chunks of arbitrary data (strings, objects) from results of database calls, API calls, or page rendering.
 type: application
-version: 0.5.3
+version: 0.5.4
```

Comment:
Technically this should be a minor version increase instead of a patch one, because it adds a feature. Am I wrong?